### PR TITLE
S3 content update

### PR DESCRIPTION
**Pull request by Nash0x7E2**
- Update photos and audio to support new audio enpoint

### CODE REVIEW

File: lib/backend/repositories/user_repo.dart [hunk 1]
```diff
@@ -151,6 +155,13 @@ class UserRepo {
   Future<UserProfile> updateUser(
       Map<String, dynamic> user, String userAddress) async {
     final result = await _userService.updateUser(user, userAddress);
+    final box = await Hive.openBox(HiveBoxes.kAppBox);
+    box.put(HiveKeys.kUserData, jsonEncode(result));
+    final Map<String, dynamic> decodedUserData =
+        jsonDecode(await box.get(HiveKeys.kUserData));
+    decodedUserData['user'] = result;
+    box.delete(HiveKeys.kUserData);
+    box.put(HiveKeys.kUserData, jsonEncode(decodedUserData));
```
Comment (orestesgaolin) [inline lib/backend/repositories/user_repo.dart:164]: Why putting this data twice?

File: lib/backend/services/auth_service.dart [hunk 2]
```diff
@@ -27,7 +30,8 @@ class AuthenticationServiceCentralized implements AuthenticationService {
     if (response.statusCode == 200) {
       final String authorization = response.headers['authorization'];
       final box = await Hive.openBox('app', encryptionKey: key);
-      box.put("auth", authorization);
+      await box.put("auth", authorization);
```
Comment (orestesgaolin) [inline lib/backend/services/auth_service.dart:33]: Explicit key instead of `HiveKeys`

File: lib/backend/repositories/user_repo.dart (hunk 1 above)
Comment (Nash0x7E2) [reply lib/backend/repositories/user_repo.dart:164]: We’re not, we’re updating a key on the object.

File: lib/backend/services/auth_service.dart (hunk 2 above)
Comment (Nash0x7E2) [reply lib/backend/services/auth_service.dart:33]: Good catch, thanks

**Pull request merged**